### DISCUSSION TITLE
How can I show pdf files from File?

### DISCUSSION THREAD

**Discussion by naibyaguilar** [afreakyelf/Pdf-Viewer#8] [Q&A]
I have tried using, launchPadFromPath but I want to get a pdf that is in the phone storage.

Thanks for your work

**Comment by afreakyelf** [comment]
Hi, Thanks for checking out the library.
This library does not allow the user to copy text from pdf or function as Adobe pdf.. why because the implementation of this library is in such way. The intention was to make a small library which takes lesser space for say.. sensitive documents or receipts maybe.
Launching pdf from phone storage is not the motive behind this library. 

TDLR: I am sorry, for now, you cant launch pdf from phone storage.
